ci/cpp_linters.sh

```diff
@@ -25,7 +25,13 @@ RAPIDS_VERSION_MAJOR_MINOR="$(rapids-version-major-minor)"
 source rapids-configure-sccache
 
 # Run the build via CMake, which will run clang-tidy when CUDF_STATIC_LINTERS is enabled.
-cmake -S cpp -B cpp/build -DCMAKE_BUILD_TYPE=Release -DCUDF_STATIC_LINTERS=ON -GNinja
+
+iwyu_flag=""
+# Check if the RAPIDS_BUILD_TYPE environment variable is equal to the string nightly
```

Comment:
This feels like a ChatGPT / Copilot comment. It doesn't add a lot for me as a reader. Let's remove.
```suggestion
```

Reply:
It's a comment I wrote as an instruction to copilot to write the check for me :) agreed, I'll remove.